src/commands/utils/config.ts

```diff
@@ -92,6 +92,24 @@ export function getDraftConfig(): Errorable<DraftConfig> {
     return { succeeded: true, result: config };
 }
 
+export function getRetinaConfig(): Errorable<DraftConfig> {
```

Comment:
The return type of `DraftConfig` is a bit confusing. Maybe it could just return a string, since the object only contains a single string property.

Reply:
Actually, I see the types for these config objects are declared all over the place. As a first step towards standardizing them, maybe create a type in this file, like:
```typescript
export type BinaryDownloadConfig = {
    releaseTag: string;
}
```

The return type of this function could then be `Errorable<BinaryDownloadConfig>`.

Later on, we can refactor existing config items, like the Kustomize config as:
```typescript
export type KustomizeConfig = BinaryDownloadConfig & {
    repoOrg: string;
    containerRegistry: string;
    imageVersion: string;
}
```